src/liballoc/boxed.rs

```diff
@@ -511,3 +512,53 @@ impl<'a,A,R> FnOnce<A> for Box<FnBox<A,Output=R>+Send+'a> {
 }
 
 impl<T: ?Sized+Unsize<U>, U: ?Sized> CoerceUnsized<Box<U>> for Box<T> {}
+
+impl<T: Clone> Clone for Box<[T]> {
```

Comment:
Oh I guess we add stability markers to these now

`#[stable(feature = "boxed_slice_clone", since = "1.3.0")]`